client/package.json

```diff
@@ -58,7 +58,7 @@
     "zone.js": "^0.8.19"
   },
   "devDependencies": {
-    "@angular/cli": "6.0.0-beta.4",
+    "@angular/cli": "1.7.3",
```

Comment:
It also works with `6.0.0-beta.6`. Don't we wanna upgrade instead of downgrade?

Reply:
The installation error will be fixed with 6.0.0-beta.6 onwards, but it'll break runtime with beta.8. This is due to APF and CLI Architect, so I'd rather go down to stable and upgrade once 6 is stable as well (cuz it doesn't seem to be right now)

Reply:
Got it.